crates/fuzzing/Cargo.toml

```diff
@@ -16,6 +16,7 @@ wasmparser = "0.70"
 wasmprinter = "0.2.17"
 wasmtime = { path = "../wasmtime" }
 wasmtime-wast = { path = "../wast" }
+wasm-encoder = "0.1"
```

Comment:
Heads up: we just published 0.2 recently, and may have 0.3 out in the next day or so.